d4s/src/main/scala/d4s/codecs/D4SAttributeEncoder.scala

```diff
@@ -80,5 +80,14 @@ object D4SAttributeEncoder {
       item.map(T.encodeAttribute).getOrElse(AttributeValue.builder().nul(true).build())
   }
 
+//  implicit def eitherEncoder[A: D4SAttributeEncoder, B: D4SAttributeEncoder](keyLeft: String, keyRight: String): D4SAttributeEncoder[Either[A, B]] = {
```

Comment:
@vpodlnk Disregard that, because derived just calls Magnolia.gen you can just derive this instance:
```scala

  implicit def eitherEncoder[A: D4SAttributeEncoder, B: D4SAttributeEncoder]: D4SEncoder[Either[A, B]] = D4SEncoder.derived
```

Reply:
So, remove these custom key params? Now we cant derive Either without this encoder.
Or maybe, I didn't understand you correctly.
```scala
implicit def eitherEncoder[A: D4SAttributeEncoder, B: D4SAttributeEncoder]: D4SEncoder[Either[A, B]] =
  D4SEncoder.noncastedGen[Either[A, B]].asInstanceOf[D4SEncoder[Either[A, B]]]
```

Reply:
@vpodlnk Yes, remove, there's no reason to name the branches anything other than "Left" and "Right".
Why can't derive? This compiled for me on this branch:
```scala
  implicit def eitherEncoder[A: D4SAttributeEncoder, B: D4SAttributeEncoder]: D4SAttributeEncoder[Either[A, B]] =
    D4SAttributeEncoder.derived
```

I would assume this would work fine with `D4SEncoder` too and the [manual cast](https://github.com/PlayQ/d4s/pull/37#discussion_r412437382) trick above

Reply:
@neko-kai, I'm sorry, I've made a type and haven't spotted it quickly